### PR TITLE
Checkout at version

### PR DESCRIPTION
**Pull request by jgarberdds**
Pull down the tagged release (or master) for each service specified in `services.ini`

### CODE REVIEW

File: script/bootstrap [hunk 1]
```diff
@@ -8,17 +8,25 @@
...
-    git clone -q -o github github.com:dod-ccpo/$service
+    echo "Cloning $service at $version"
+    curl -sSL https://github.com/dod-ccpo/$service/archive/$version.tar.gz | tar xvz
+    mv $service-$version $service
     # "script/setup is typically run after an initial clone"
     cd $top/$service
     log=$top/log/$service.setup.log
     echo "Writing to $log";
-    script/setup > $log 2>&1 || echo "Setup failed for $service"
+    script/setup 2>&1 | tee $log || echo "Setup failed for $service"
```
Comment (briandds) [inline script/bootstrap]: I would say leave out the `tee` -- there is way too much noise in the terminal.  You could put a message about how the user can run `tail -f $log` if they want to see it all.

**Review by briandds — DISMISSED** [review]
Great -- I ran it locally and it worked.  Creative use of shell arrays and `IFS` :)  Just a few requests for changes to cut down on the verbosity -- I would say these scripts should produce concise meaningful human readable output but with information about where to find logs for details.

File: script/bootstrap [hunk 2]
```diff
@@ -8,17 +8,25 @@
...
+    IFS="@"
+    set $repo
+    service=$1
+    version=$2
+
     cd $top
-    echo "Cloning $service"
-    git clone -q -o github github.com:dod-ccpo/$service
+    echo "Cloning $service at $version"
+    curl -sSL https://github.com/dod-ccpo/$service/archive/$version.tar.gz | tar xvz
```
Comment (briandds) [inline script/bootstrap]: Also, I'd say leave out the `v` option to cut down on the noise (or redirect the output to a log file)

**Comment by jgarberdds** [comment]
@briandds How's this? less verbose output

**Review by briandds — APPROVED** [review]

**Pull request merged**